docs/architecture/adr-016-protocol-versions.md

```diff
@@ -268,12 +282,12 @@ help old peers, possibly on older versions, sync the blockchain.
 We might be tempted to say that we only connect to peers with the same
 AppVersion and BlockVersion (since these define the consensus critical
 computations), and a select list of P2PVersions (ie. those compatible with
-ours), but then we'd need to make accomodations for connecting to peers with the
+ours), but then we'd need to make accommodations for connecting to peers with the
 right Block/AppVersion for the height they're on.
 
 For now, we will connect to peers with any version and restrict compatibility
 solely based on the ChainID. We leave more restrictive rules on peer
-compatibiltiy to a future proposal.
+compatibility to a future proposal.
```

Comment:
Is now the time we want to add any more version compatibility checks between peers?

Reply:
Phase 2, which is when we'll consider significant changes to the actual wire protocol. For now we're trying to keep the current protocol as far as possible.

Reply:
cool :+1:  I'll leave this here then